README.md

```diff
@@ -18,7 +18,30 @@ This [demo](https://czeckd.github.io/angular-svg-icon/) shows this module in act
 ```
 $ npm i angular-svg-icon --save
 ```
+
+## Versions
+
+The latest version of the package is for Angular 18.
+
+
+:grey_exclamation: **BREAKING CHANGE**: as of angular-svg-icon@18.0.0, the package was converted to use 
+`inject` and `signal` from `@common/core` for improved performance. 
+
+This may potentially break usage of two-way binded variables. In brief, an application using this 
+library may need to be changed to use signals in order to use version 18 and beyond. This should 
+be a simple matter of converting plain old variables that are 2-way binded into signals. For example:
+
+```typescript
+  a = 120;
+```
+should be modified to become:
+```typescript
+  a = signal(120);
```

Comment:
No, a =120 is usable with signal. 

Only breaking change is avoiding method calls for inputs. "Please mention only that avoid method calls in inputs"

Reply:
@sheikalthaf - is the revised wording correct?

Reply:
@czeckd yes